### PR TITLE
V.3.0.0

### PR DESCRIPTION
**Pull request by v-ruboranov**

### CODE REVIEW

File: capabilities.json [hunk 1]
```diff
@@ -716,64 +610,55 @@
...
     },
-    "supportsMultiVisualSelection": true
+    "supportsMultiVisualSelection": true,
+    "supportsHighlight": true,
+    "supportsKeyboardFocus": true,
+    "privileges": [
+        {
+            "name": "WebAccess",
+            "essential": true,
+            "parameters": ["https://*", "ftp://*"]
```
Comment (uve) [inline capabilities.json:661]: remove ftp

**Review by uve — CHANGES_REQUESTED** [review]

File: capabilities.json (hunk 1 above)
Comment (v-ruboranov) [reply capabilities.json]: resolved in this commit https://github.com/microsoft/powerbi-visuals-enhancedscatter/pull/93/commits/a70c2ed983577cab14d4bb990aad63ea9827608e

File: pbiviz.json [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
     "guid": "EnhancedScatterChart1443994985041",
     "visualClassName": "EnhancedScatterChart",
-    "version": "2.0.1",
+    "version": "3.0.0",
     "description": "A few more properties were added to the existing scatter chart visual, including shapes as markers, background image support, and developer crosshairs for positioning elements onto an image background.",
     "supportUrl": "https://community.powerbi.com",
     "gitHubUrl": "https://github.com/Microsoft/powerbi-visuals-enhancedscatter"
   },
-  "apiVersion": "3.2.0",
+  "apiVersion": "5.1.0",
```
[the remark this answers is not in the document]
Comment (v-ruboranov) [reply pbiviz.json]: resolved here https://github.com/microsoft/powerbi-visuals-enhancedscatter/pull/93/commits/a70c2ed983577cab14d4bb990aad63ea9827608e

File: src/EnhancedScatterChart.ts [hunk 3]
```diff
@@ -769,12 +768,14 @@ export class EnhancedScatterChart implements IVisual {
...
 
+        this.hasHighlights = dataValues.length > 0 && dataValues.some(value => value.highlights && value.highlights.some(_ => _));
+
         const sizeRange: ValueRange<number> = EnhancedScatterChart.getSizeRangeForGroups(
             grouped,
             scatterMetadata.idx.size
         );
 
-        settings.fillPoint.isHidden = !!(sizeRange && sizeRange.min);
+        settings.enableFillPointCardSettings.IsHidden = !!(sizeRange && sizeRange.min);
```
[the remark this answers is not in the document]
Comment (v-ruboranov) [reply src/EnhancedScatterChart.ts]: fixed here https://github.com/microsoft/powerbi-visuals-enhancedscatter/pull/93/commits/7c93ec3cd0f6e4987f66ce7b68bf07870037d1bc

File: capabilities.json [hunk 4]
```diff
@@ -474,40 +464,28 @@
             }
         },
         "categoryAxis": {
-            "displayName": "X-Axis",
```
[the remark this answers is not in the document]
Comment (v-ruboranov) [reply capabilities.json:477]: All settings are moved to EnhancedScatterChartSettingsModel.ts file, and configured there.

File: src/EnhancedScatterChart.ts [hunk 5]
```diff
@@ -1543,21 +1560,23 @@ export class EnhancedScatterChart implements IVisual {
...
     }
 
     private adjustViewportByBackdrop(): void {
-        const img: HTMLImageElement = new Image(),
-            that: EnhancedScatterChart = this;
+        const img: HTMLImageElement = new Image();
 
-        img.src = this.data.settings.backdrop.url;
+        // eslint-disable-next-line
+        const that: EnhancedScatterChart = this;
```
[the remark this answers is not in the document]
Comment (v-ruboranov) [reply src/EnhancedScatterChart.ts]: for accessing outside context (https://github.com/microsoft/powerbi-visuals-enhancedscatter/pull/93/commits/6c9ba830b8a77cc98cbe9e2d25f0379957ea1a9b)

**Pull request merged**